src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -221,6 +222,9 @@ public long StartBackupTask(long taskId, bool isFullBackup, long? operationId =
                 throw new InvalidOperationException($"Backup task id: {taskId} doesn't exist");
             }
 
+            if (operationId == AbstractOperations<Operations.Operation>.InvalidOperationId)
+                return periodicBackup.RunningTask != null ? periodicBackup.RunningTask.Id : AbstractOperations<Operations.Operation>.InvalidOperationId;
```

Comment:
it may be racy IIRC, before checking if Running task is not null and returning id, the task might be set to null, introduce a variable `var runningTask = periodicBackup.RunningTask` to avoid such a situation

Reply:
done